### ISSUE TITLE
"Resuming" same as just download from a ckpt path?

### ISSUE THREAD

**Issue by animeking527** [TheLastBen/fast-stable-diffusion#435]
When I try to load a previous session everything appears to look right (model downloads, instance img folder loads, etc), but when I run the training cell with resume training checked, it just instantly goes to "converting ckpt. Done. Find it in your gdrive"

So I just put the path directly to the last ckpt from that session in the download model cell, kept resume training checked and it appears to be working now.

**Comment by TheLastBen** [comment]
are you using the latest colab ? if there is an error in training, it might show "converting ckpt. done". immediately

**Comment by TheLastBen** [comment]
remove this part in the training cell, and resume training to display the error message

![s](https://user-images.githubusercontent.com/110583491/200611214-c9d0e66a-6568-4a53-a3ed-fdbb8a7113ac.JPG)

**Comment by animeking527** [comment]
Yep, latest colab. Also curious why it tries to link to huggingface with the session name? Is it actually sending anything there or it is just a default message?

**Comment by animeking527** [comment]
Also, training off off the ckpt path appears to have worked this time. 

The first time I did it (og ckpt 7500 steps + 500 for the new training) the images became really noisy/glitchy at my benchmark (30steps, 10cfg, dpm++2m) for some reason (though looking good around 60 steps).

This time (og ckpt 6000steps + 2000 new steps) and the outputs appear as expected.

**Comment by TheLastBen** [comment]
the error you're getting is because you set "contain_faces" to male, female or "both", and you disabled 'text_encoder_training'.

**Comment by TheLastBen** [comment]
> Yep, latest colab. Also curious why it tries to link to huggingface with the session name? Is it actually sending anything there or it is just a default message?

what do you mean ?

**Comment by animeking527** [comment]
Hmm. I did have it set to "female" (how it was set in the initial training). Should this be different (set to "no") when loading & resumeing?

I made sure to kept the text encoder on (previously I didn't, thinking thats what causes those noisy outputs from before)

> OSError: /content/models/wkwphto3 does not appear to have a file named text_encoder_trained/config.json. Checkout 'https://huggingface.co//content/models/wkwphto3/None' for available files.

**Comment by TheLastBen** [comment]
the last colab update was 5 hours ago, make sure you got it : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Comment by animeking527** [comment]
Fresh colab still getting the same error even with faces checked to no & text encoder on. 

As I mentioned is appears just putting the specific checkpoint in the ckpt path and having resume checked on training cell does work (both with faces set to 'female' or 'no'). 

Is this exactly the same as the intended method? I don't know, but the outputs appear to look like they're continuing where the training left off.

**Comment by TheLastBen** [comment]
don't use the ckpt path to resume, it could get confusing, share your session gdrive folder and use that link in the session cell

**Comment by animeking527** [comment]
Hmm, doing so said there was no model found (permissions set correctly on the folder). 

> Downloading session...
Loading session with no previous model, using the original model or the custom downloaded model
Insert your huggingface token :
